site/content/en/reference/live/apply/_index.md

```diff
@@ -91,6 +91,47 @@ is added to the name. Example:
 test-grouping-object-17b4dba8
 ```
 
+### Ordering
+
+`kpt live apply` will sort the resources before applying them. This makes sure
+namespaces are applied before resources going into the namespace, configmaps
+are applied before Deployments and StatefulSets that might reference them, and
```

Comment:
"Deployments and StatefulSets that might reference them"

On first read, this sounds like you're inspecting Deployments for refs to ConfigMaps. But later on, it seems the ordering is purely based on kind and not their contents. May want to clarity this.

Reply:
I have updated the text to make this clearer.